### ISSUE TITLE
Plugin creates series with empty title?

### ISSUE THREAD

**Issue by KatrinIhler** [opencast-ilias/OpenCast#328]
We've seen two institutions so far with series that have empty titles in their Opencast, which leads to problems. Opencast itself doesn't allow the creation of events or series with empty required fields via it's UI, but afaik the External API doesn't check those requirements. Both of the affected institutions use the ILIAS plugin for integration with Opencast. It stands to reason that the plugin allows the creation of series with empty titles, although I haven't confirmed this suspicion yet.

See also https://github.com/opencast/opencast/issues/6032.

**Comment by reiferschris** [comment]
Hi, at least currently this is not possible. Title was at least to my knowledge always required in the plugin.
I would assume that you could check which user created those series? At least in our systems we use a dedicated user for the plugin which is also in the metadata field 'createdBy' of the series.

**Comment by dagraf** [comment]
Thanks for reporting. I just did some testing. Here my findings:
- When in the Opencast plugin the tab "Meta data" and there the subtab "Series" is properly configured (see screenshot) it is not possible to create a series without a title. Because the field "Title" is always "Required" as soon as it is added in the configuration of the plugin.
- When the metadata field "Title" is not configured at all (for example because I deleted it in the configuration) and I try to create an series, I get an error message. But the series in Opencast gets created anyhow. 

I would suggest we discuss in our next community meeting how to solve this problem.

**Possible correct configuration:**
<img width="799" alt="Bildschirmfoto 2024-07-29 um 15 14 57" src="https://github.com/user-attachments/assets/2a10472a-ba4c-4d56-baff-97eb55c79912">

**Error message:**
<img width="1558" alt="Bildschirmfoto 2024-07-29 um 15 25 03" src="https://github.com/user-attachments/assets/3b28c83c-dcbd-4ce8-98ab-491687f32635">
